packages/cactus-plugin-odap-hermes/README.md

```diff
@@ -123,6 +123,10 @@ A test with a rollback after a timeout (client crashed):
 
 https://github.com/hyperledger/cactus/blob/main/packages/cactus-plugin-odap-hermes/src/test/typescript/integration/odap/odap-rollback.test.ts
 
+A test with a backup gateway resuming the protocol after the client gateway crashed:
+
+https://github.com/hyperledger/cactus/blob/main/packages/cactus-plugin-odap-hermes/src/test/typescript/integration/odap/backup-gateway-after-client-crash.test.ts
```

Comment:
@AndreAugusto11 This link will die if the file gets renamed, moved or deleted on the main branch in the future. What I recommend instead is getting the direct link to the current latest commit on main (the head) and then link to the file through that. 
This way it doesn't matter what happens on the main branch, the link will be alive as long as GitHub is alive (which is also a risk we are taking, but an exponentially lower one compared to the first one I explained above - at least according to my own estimates)

Reply:
It makes sense. The only one pointing to the main branch is the `odap-rollback.test` since I can only know the commit hash after the PR is merged